### PR TITLE
Add URL to all pom files

### PR DESCRIPTION
**Pull request by huehnerlady**
#214  unfortunately did not change anything, I got the help from gradle and they suggested this to add the url to all the pom files

### CODE REVIEW

**Review by NilsDammenhayn — APPROVED** [review]
Das Log soll da bleiben?

File: build.gradle.kts [hunk 1]
```diff
@@ -79,11 +79,10 @@ gradlePlugin {
     }
   }
   publishing {
-    publications {
-      register("pluginMaven", MavenPublication::class) {
-        pom {
-          url.set("https://github.com/europace/docker-publish-gradle-plugin")
-        }
+    publications.withType(MavenPublication::class).configureEach {
+      logger.lifecycle("publication: $name")
```
Comment (huehnerlady) [inline build.gradle.kts:83]:
```suggestion
```

**Comment by huehnerlady** [comment]
> Das Log soll da bleiben?

nö 😅

**Review by NilsDammenhayn — APPROVED** [review]
👌

**Pull request merged**